rclpy/__init__.py.in

```diff
@@ -27,7 +27,8 @@ del source_path
 
 for __execfile in __execfiles:
     with open(__execfile, 'r') as __fh:
-        exec(__fh.read())
+        code_object = compile(__fh.read(), __execfile, 'exec')
```

Comment:
The local variable `code_object` should be prefixed with `__` as well as deleted after its usage scope.